static/scripts/search.js

```diff
@@ -1,14 +1,17 @@
 $( document ).ready(function() {
+    var searchAttr = 'search-mode';
     jQuery.expr[':'].Contains = function(a,i,m){
         return (a.textContent || a.innerText || "").toUpperCase().indexOf(m[3].toUpperCase())>=0;
     };
     //on search
     $("#nav-search").on("keyup", function(event) {
         var search = $(this).val();
+
         if (!search) {
             //no search, show all results
-            $("nav > ul > li").show();
-            
+            document.documentElement.removeAttribute(searchAttr);
```

Comment:
Would it be possible to better user custom attributes through `data-*` as it would be validated and is part of HTML5 specification
Or using it as additional class and overriding in CSS would work too :)

Reply:
Done.